packages/runtime-vapor/src/index.ts

```diff
@@ -50,6 +50,7 @@ export {
   type FunctionalComponent,
   type SetupFn,
 } from './component'
+export { withCtx } from './componentRenderContext'
```

Comment:
Should it be exported? Or only for internal usage?

Reply:
Only used within the render function

Reply:
The relevant code has been deleted during refactoring